### ISSUE TITLE
Mist --shh

### ISSUE THREAD

**Issue by segovro** [ethereum/mist#1273]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````
I would like that MIST could be launched with whisper -shh option. Now it is posible to launch geth with --shh, and then Mist. Could this be included in the Mist browser itsefl?

**Comment by luclu** [comment]
@segovro please try the `--node-shh` flag:
> Node options:
  -  To pass options to the underlying node (e.g. Geth) use the --node- prefix,
     e.g. --node-datadir

**Comment by segovro** [comment]
thanks, will try

**Comment by luclu** [comment]
I played all little with this and it seems that simple flags will be passed **and** appended by a `true` to the node. It seems that not all flag like `--rpc` will work this way.
Could you enlight us, @hiddentao?

**Comment by hiddentao** [comment]
`--rpc` is a special flag which cannot be passed onto the geth node since Mist itself uses that flag.  As for `true` being appended I have addressed that now in #1283
